arches/app/templates/views/components/etl_modules/bulk_edit_concept.htm

```diff
@@ -0,0 +1,280 @@
+{% extends "views/components/etl_modules/base-import.htm" %}
+{% load i18n %}
+
+{% block task_details %}
+<div class="etl-module-component-container">
+    <div class="etl-module-body">
+        <h2>
+            <span style="font-weight: 400; font-size: 1.7rem; color: #565656;">Replace Concept</span>
```

Comment:
It looks like this and several other strings in this template could be in trans tags.

Reply:
Good catch!

Reply:
Ok, those items should be resolved @chiatt